02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -11,34 +11,53 @@ output:
 
 # Purpose of this analysis
 
-This example is one of pathway analysis module set, we recommend looking at the [pathway analysis introduction](https://alexslemonade.github.io/refinebio-examples/02-microarray/pathway-analysis_microarray_00_intro.html) to help you determine which pathway analysis method is best suited for your purposes.
+This example is one of pathway analysis module set, we recommend looking at the [pathway analysis table below](#how-to-choose-a-pathway-analysis) to help you determine which pathway analysis method is best suited for your purposes.
 
 This particular example analysis shows how you can use over-representation analysis (ORA) to determine if a set of genes (e.g., those differentially expressed using some cutoff) shares more or fewer genes with gene sets/pathways than we would expect at random.
 This pathway analysis method does not require any particular sample size, since the only input from your dataset is a set of genes of interest [@Yaari2013].
 
 ⬇️ [**Jump to the analysis code**](#analysis) ⬇️
 
+### What is pathway analysis?
+
+We refer to any technique that uses predetermined sets of genes that are related or coordinated in their expression in some way (e.g., participate in the same molecular process, are regulated by the same transcription factor) to interpret a high-throughput experiment as pathway analysis.
+In the context of refine.bio, we use these techniques to analyze and interpret genome-wide gene expression experiments.
```

Comment:
We link to refine.bio further down in the notebook at line 114 (probably because its relevance to that section but also perhaps because it was the first instance of refine.bio being mentioned in the context of the notebook).

Now that this is the first time mentioning refine.bio I think that we should link to the webpage here, what do you think?